tools/webcam/camera.py

```diff
@@ -1,16 +1,19 @@
 import av
+import platform
 
 class Camera:
   def __init__(self, cam_type_state, stream_type, camera_id):
-    try:
-      camera_id = int(camera_id)
-    except ValueError: # allow strings, ex: /dev/video0
-      pass
     self.cam_type_state = cam_type_state
     self.stream_type = stream_type
     self.cur_frame_id = 0
 
-    self.container = av.open(camera_id)
+    print(f"opening {cam_type_state} at {camera_id}")
+
+    if platform.system() == "Darwin":
+      self.container = av.open(camera_id, format='avfoundation', container_options={"framerate": "20"})
```

Comment:
can we just do this on all platforms?

Reply:
not sure, format='avfoundation' is only for mac. will see what I can do.